### PR TITLE
:sparkles: 186 number input component

### PR DESCRIPTION
**Pull request by lehju**
**Description**

Add component to select a number with buttons

**Reference**

Issues #186

### CODE REVIEW

File: src/components/Form/MucCounter.vue [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      v-on:click="clickedMinus"
+      variant="secondary"
+      :disabled="disableMinus"
+    >
+      <template #default><muc-icon icon="minus" /></template>
+    </MucButton>
+    <p>
+      <strong
+        class="centered-text"
+        style="color: #005a9f"
```
Comment (FabianWilms) [inline src/components/Form/MucCounter.vue:13]: You can use this variable instead: https://github.com/it-at-m/muc-patternlab-vue/blob/beta/public/assets/temporary/custom-style.css#L6

File: src/components/Form/MucCounter.vue [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+/**
+ * Input value from the counter component.
+ */
+const modelValue = defineModel<number>({ default: 0 });
+
+const props = withDefaults(
+  defineProps<{
+    /**
+     * Label shown behind the counter
```
Comment (FabianWilms) [inline src/components/Form/MucCounter.vue:57]:
```suggestion
     * Label shown after the counter
```

**Review by FabianWilms — CHANGES_REQUESTED** [review]
Rest looks good!

File: src/components/Form/MucCounter.vue (hunk 1 above)
Comment (lehju) [reply src/components/Form/MucCounter.vue]: Done.

**Review by FabianWilms — APPROVED** [review]

**Pull request merged**